docs/synopsis.md

```diff
@@ -203,6 +203,10 @@ This method implements logic of mapping `source` to `to_table` and allows custom
 You can use one `source` for a lot of `to_table`.
 
 The following options are available inside the block:
+- `on_destroy` - defines a custom logic and behavior for `destroy` event:
+  - definition: `on_destroy { |attributes:|, |target_keys:| ...your code... }`
```

Comment:
```ruby
on_destroy { |attributes:|, |target_keys:| ...your code... }
```
This doesn't look like a valid ruby code to me :(

Reply:
oops :)